### PR TITLE
Updates to Neo4j Desktop related pages

### PR DESCRIPTION
**Pull request by adam-cowley**
- [x] Updated screenshots on /developer/aura-connect-neo4j-desktop/ : #211
- [x] Update screenshots and copy on /developer/neo4j-desktop/
- [x] New video for Neo4j Desktop

### CODE REVIEW

File: aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc [hunk 1]
```diff
@@ -48,22 +48,26 @@ image::{img}3desktopConnect_remote_graph.jpg[link="{img}3desktopConnect_remote_g
...
 [start=7]
-7. If the connection succeeds, we will see our database name and url details. To activate the connection, click the `Activate` button in lower right. Once activated, we can deactivate with the same button or connect to the instance through Neo4j Browser using the button on the left.
+7. If the connection succeeds, we will see our database name and url. To activate the connection, click the `Connect` button in lower right.
+Once activated, we can disconnect with the same button, open Neo4j Browser with the `Open` button to the left or open a https://neo4j.com/developer/graph-app-development/[Graph App^] using the dropdown menu next to the Open button.
 
 image::{img}7desktopConnect_connected_remote.jpg[link="{img}7desktopConnect_connected_remote.jpg",role="popup-link"]
 
-Note: Deactivating in Neo4j Desktop will not shut down or stop the remote instance - only temporarily close the connection to it in Neo4j Desktop.
-Note: Neo4j Desktop only allows 1 connection at a time to a database (local or remote).
+**Note:** Deactivating in Neo4j Desktop will not shut down or stop the remote instance - only temporarily close the connection to it in Neo4j Desktop.
```
Comment (jexp) [inline aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc:74]: you can use NOTE: or for a block [NOTE]

File: aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc [hunk 2]
```diff
@@ -48,22 +48,26 @@ image::{img}3desktopConnect_remote_graph.jpg[link="{img}3desktopConnect_remote_g
 image::{img}4desktopConnect_name_url.jpg[link="{img}4desktopConnect_name_url.jpg",role="popup-link"]
 
 [start=5]
-5. Now we need to choose an authentication method. For this example, we will use the standard username and password method. Choose that option.
+5. Now we need to choose an authentication method. For Neo4j Aura, we need to choose `Username/password`.
```
Comment (jexp) [inline aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc]: select or pick for the 2nd choose ?

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 3]
```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
 [#desktop-activate]
 == Activation
 
-image::{img}desktop_registration_dec2019.jpg[link="{img}desktop_registration_dec2019.jpg",float="right"]
+[#desktop-first-time]
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc]: [[desktop-first-time]]

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 4]
```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
...
 If you are opening Neo4j Desktop for the first time, it should ask you to register the software with an activation key.
-When you downloaded Neo4j Desktop, your unique activation key was displayed at the top of the instructions page.
+This activation key is generated when you first download Neo4j Desktop and will be displayed on the download confimration page.
 While we can copy it to the clipboard and paste it into the provided input in Neo4j Desktop, we also recommend copying it to a safe place as a backup (local file, password vault, or another method).
 
+image::{img}desktop_registration_may2020.jpg[link="{img}desktop_registration_may2020.jpg",float="right"]
+
+Copy this activation key into the Software regisration form and click Activate to continue.
+
+As a new user to Neo4j Desktop, you will be greated with a new sample database containing https://sandbox.neo4j.com/?usecase=recommendations[Movies and Actors].
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc]: not sure if the sandbox link here is rather distracting, perhaps rather link to 

https://github.com/neo4j-graph-examples/movies

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 5]
```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
...
+
+image::{img}first_graph_experience.png[link="{img}first_graph_experience.png",float="right"]
+
+Clicking the Browser Guide link for the Sample database will open up the Neo4j Browser with a Browser Guide that will guide you through your first queries on this Neo4j database.
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc:62]: the password thingy?

The sample database has a randomly generated password, that you can override in the Administration pane of the Manage Database view.

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 6]
```diff
@@ -44,21 +44,28 @@ Step 2 on the download page instructions shows you how to set up your first proj
...
+
 [#desktop-navigate]
 == Navigation
 
 The menu along the left side of Neo4j Desktop shows high-level icons for projects, graph applications, settings, activation keys, and Neo4j info.
 At the top, the folder with a bookmark (image:{img}projects_icon.jpg[width=20]) shows the list of projects you have created.
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc:71]: I wonder if these icons can be found somewhere in fontawesome. But that's not critical :) for the publication. Just curiousity.

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 7]
```diff
@@ -100,38 +109,52 @@ A panel will display on the right to show the databases in your project and the
 
 To create a database, click in the dotted square that says `Add Graph`.
 Then click the kbd:[Create a Local Graph] button.
-Next, you will type in a name for your database and set a password and choose a version (default is latest version).
-A small icon will show which versions have been downloaded already.
+Next, you will type in a name for your database set a password.  You can also choose a specific version of neo4j (the default is latest version).
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc]: Neo4j capitalized

Comment (jexp) [reply graph-platform/neo4j-desktop/neo4j-desktop.adoc]: One sentence per line :)

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 8]
```diff
@@ -100,38 +109,52 @@ A panel will display on the right to show the databases in your project and the
...
+A small icon next to each version number in the dropdown menu will show which versions have been downloaded already.
 Once the info is filled in, then click kbd:[Create].
 It may take a few minutes to download the version and actually create the database.
 
 ****
 [NOTE]
-Please remember your database password. 
+Please remember your database password.
 If you forget it, you will have to remove the credentials file in `data/dbms/authorization`.
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc]: Override password in the Administration pane instead.

Comment (jexp) [reply graph-platform/neo4j-desktop/neo4j-desktop.adoc]: Also as most graph apps work automatically without the password, this is only relevant for driver access, so perhaps we can deemphasise?

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc [hunk 9]
```diff
@@ -100,38 +109,52 @@ A panel will display on the right to show the databases in your project and the
...
 
+The kdb:[Start] button in the bottom right hand corner of each database card will Start the database.
+Once started, there will be a button in the bottom right hand corner to Stop the database.
+
+Once a database has been started, the Open button in the bottom left hand corner of the card will be enabled.
+Clicking Open will open link:/developer/neo4j-browser/[Neo4j Browser^] and connect to the active database.
+Next to the Open button is a dropdown menu which will allow you to open up a list of link:/developer/graph-app-development[Graph Apps^].
+
+
 [#desktop-manage-db]
```
Comment (jexp) [inline graph-platform/neo4j-desktop/neo4j-desktop.adoc]: [[desktop-manage-db]]

**Comment by jexp** [comment]
- can you add a link to the desktop category on community.neo4j.com instead of the mention of intercom
- and a link to the desktop GH issues. 

Also a short section on graph apps? What they are and how to install/start them? Perhaps metnion bloom as pre-installed graph app and link to the bloom guide?

Otherwise looks great.

**Review by jexp — CHANGES_REQUESTED** [review]
Some small changes, otherwise looks great.

File: aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc (hunk 2 above)
Comment (adam-cowley) [reply aura-cloud-dbaas/aura-neo4j-desktop/aura-neo4j-desktop.adoc]: Agreed, this isn't the best sentence.  I've updated to:
```
5. Now we need to choose an authentication method.  
Neo4j Aura instances require username and password authentication, so click the `Username/password` button.
```

File: graph-platform/neo4j-desktop/neo4j-desktop.adoc (hunk 6 above)
Comment (adam-cowley) [reply graph-platform/neo4j-desktop/neo4j-desktop.adoc:71]: Good question, I assume they are Semantic UI.  I can look at including these during the redesign

**Review by JMHReif — APPROVED** [review]

**Pull request merged**